mlfinlab/bet_sizing/ch10_snippets.py

```diff
@@ -0,0 +1,129 @@
+"""
+This module contains the code snippets found in Chapter 10 of
+"Advances in Financial Machine Learning" by Marcos López de Prado.
+The code has been ammended for readability, to conform to PEP8
+rules, to keep the snippets as manageable single-units of
+functionality, as well as to account for depracation of functions
+originally used, but is otherwise unaltered.
+"""
+
+
+# imports
+import pandas as pd
+from scipy.stats import norm
+from mlfinlab.util.multiprocess import mp_pandas_obj
+
+
+def get_signal(prob, num_classes, pred=None):
+    """
+    SNIPPET 10.1 - FROM PROBABILITIES TO BET SIZE
+    Calculates the given size of the bet given the side and the
+    probability (i.e. confidence) of the prediction. In this
+    representation, the probability will always be between
+    1/num_classes and 1.0.
+
+    :param prob: (pd.Series) The probability of the predicted bet side.
+    :param num_classes: (int) The number of predicted bet sides.
+    :param pred: (pd.Series) The predicted bet side. Default value is None
+        which will return a relative bet size (i.e. without multiplying
+        by the side).
+    :return: (pd.Series) The bet size.
+    """
+    # get signals from predictions

```

Comment:
Inline comments start with a Capital letter(and end with a full stop, where applicable). This is in line with keeping a uniform format throughout our code base. If this is against PEP8 standard, please let me know.

Reply:
Will do.